### DISCUSSION TITLE
快一點？

### DISCUSSION THREAD

**Discussion by chia-kai-chang** [1101-GS4719-C/Python-Community-GS4719-C#50] [閒聊八卦]
![image](https://user-images.githubusercontent.com/1768891/135750150-97c2124f-72d4-4038-a829-a4faed1b066b.jpeg)

**Comment by enoch-lightning** [comment]
That's quite interesting! He uses very powerful devices to run the simplest code.我其實不太清楚CPU、RAM與NVME DRIVE是什麼與功能為何，但我想這篇漫畫的幽默，應在於「殺雞用牛刀。」

**Comment by enoch-lightning** [comment]
:smile:
